airflow/configuration.py

```diff
@@ -588,7 +588,7 @@ def parameterized_config(template):
     )
 
 
-WEBSERVER_CONFIG = AIRFLOW_HOME + '/webserver_config.py'
+WEBSERVER_CONFIG = os.path.abspath(AIRFLOW_HOME + '/webserver_config.py')
```

Comment:
Instead of here we should make AIRFLOW_HOME become an absolute path.

Reply:
Or could we give a message of this?

Reply:
I don't mind either way.